kms/yubikey/yubikey_no_cgo.go

```diff
@@ -15,6 +15,6 @@ import (
 func init() {
 	apiv1.Register(apiv1.YubiKey, func(ctx context.Context, opts apiv1.Options) (apiv1.KeyManager, error) {
 		name := filepath.Base(os.Args[0])
-		return nil, errors.Errorf("unsupported kms type 'yubikey': %s is compiled without cgo support", name)
+		return nil, errors.Errorf("unsupported kms type 'yubikey': %s is compiled without CGO or YubiKey support", name)
```

Comment:
I'm okay with `CGO`, but the Go docs use `cgo`

Reply:
Fixed with 5744559